control/veda_val.py

```diff
@@ -36,6 +34,8 @@
 from control_env import *
 from control.veda_utils import ErrorObject, Output
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
We may need to add a comment that why we need to add the log here for future references.